### PR TITLE
Pontoon: document user deactivation

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

**Comment by flodolo** [comment]
Removal would be [tracked here](https://bugzilla.mozilla.org/show_bug.cgi?id=1561663), but I know some are not fans of adding bug links to documentation.

File: src/tools/pontoon/deactivating_users.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Deactivating Pontoon users
+
+Access Django’s admin interface at `https://pontoon.mozilla.org/a/` (note that this is not the usual admin interface), then click `Users`. In the next page search for the user you want to deactivate (safer to search by email address):
+
+* Click on the user among the results displayed under the search field.
+* In the *Permissions* section, deselect the `Active` checkbox and click `SAVE` at the bottom of the page.
+
+This will prevent them from logging in in Pontoon, and disconnect them immediately from the system.
```
Comment (mathjazz) [inline src/tools/pontoon/deactivating_users.md]: I think it's grammatically (#1) and semantically (#2) more correct like this:
1. ...from logging **into** Pontoon...
1. ...and **log them out** immediately...

But I'm not sure, so feel free to ignore if you disagree.

**Review by mathjazz — APPROVED** [review]

File: src/tools/pontoon/deactivating_users.md (hunk 1 above)
Comment (gueroJeff) [reply src/tools/pontoon/deactivating_users.md]: "This will prevent them from _logging into Pontoon_, and _will log them out of the system immediately_."

Comment (flodolo) [reply src/tools/pontoon/deactivating_users.md]: Thanks, fixing it :-)

I kind of remember Jeff writing "logging in in something", so I called for help.

Comment (gueroJeff) [reply src/tools/pontoon/deactivating_users.md]: I are well English sporker ;-)

**Pull request merged**